test/utils/React-util.js

```diff
@@ -19,7 +19,7 @@ import {
 /**
  * Render a ReactElement and use helpful test utility methods against it.
  * @param {ReactElement} reactElement The element to render.
- * @returns {RenderedTree}
+ * @returns {RenderedTree} The return
```

Comment:
Oh hack...  I added that fix on the list-examples branch.  Shame on me.  I'll make a separate branch and get master working so we can rebase open PRs.